src/azure/cli/commands/arm.py

```diff
@@ -228,8 +231,7 @@ def one_required(namespace):
                      metavar='LIST KEY=VALUE')
     cmd.add_argument('properties_to_remove', '--remove', nargs='+', action=OrderedArgsAction,
                      default=[], help='Remove a property or an element from a list.  Example: '
-                     '--remove property.list <index>', metavar='LIST INDEX',
-                     validator=one_required)
+                     '--remove property.list <index>', metavar='LIST INDEX')
```

Comment:
We've used multi-word metavars for some other params, Johan was onboard with those changes

---
In reply to: [74599192](https://github.com/Azure/azure-cli/pull/687#discussion_r74599192) [](ancestors = 74599192)